src/aap_eda/core/enums.py

```diff
@@ -56,9 +56,9 @@ class ResourceType(DjangoStrEnum):
 
 
 class Action(DjangoStrEnum):
-    CREATE = "create"
-    READ = "read"
-    UPDATE = "update"
+    CREATE = "add"
```

Comment:
I am surprised that you can change the value here. Are `create` and `add` both predefined or it can be any word? Same for the rest of changes.

Reply:
Left side of the assignment are the conventional CRUD action name, but we have to change the values to adapt to the DAB RBAC naming convention. Also, note that these enums are only used for RBAC on API level and has no effect on db, etc.

Reply:
for curiosity, how does it work before the change? unless DAB supports both words.

Reply:
@bzwei We were not using any of the CRUD before except for READ, but I'm a bit confused why it was working for READ; my guess is that in all the use cases we had `detail=False` which somehow ignores or doesn't make use of `rbac_action` but setting `detail=True` and giving rbac action that doesn't match the ones in DAB result in error like `RuntimeError: The permission read_edacredential is not valid for model edacredential` so I had to make these changes above.